.idea/misc.xml

```diff
@@ -4,4 +4,5 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="JSX" />
   </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_3" assert-keyword="false" jdk-15="false" />
```

Comment:
Welche Version von Webstorm hast Du denn?

Viele Grüße
Nicole

> Am 24.04.2016 um 14:55 schrieb Alexandre Soler Sanandres <notifications@github.com>:
> 
> In .idea/misc.xml:
> 
> > @@ -4,4 +4,5 @@
> >    <component name="JavaScriptSettings">
> >      <option name="languageLevel" value="JSX" />
> >    </component>
> > +  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_3" assert-keyword="false" jdk-15="false" />
> Das wird von Webstorm jedes mal hinzugefügt. Weiss jemand Warum, bzw. wie ich es auschalte? :-) Danke
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>